forms-flow-web/src/components/Modals/ProgressBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Try using react bootstrap progress bar in this case : 
https://react-bootstrap.netlify.app/docs/components/progress